docs/example.md

```diff
@@ -0,0 +1,108 @@
+# Full workflow tutorial
+This example deploys sample aplication on OpenShift,
+exports that aplication to Nulecule, and than deploys exported applicatoin to Kubernetes.
+
+This has been tested on ADB 1.7.0 as Fedora 23 on local machine
+
+## Deploy Docker registry on your local machine
+ 1. Run Docker registry on you local machine.
+    - `docker run -d -p 5000:5000 --name registry registry:2`
+
+
+## Start ADB and deploy sample OpenShift application
+ 1. Clone the ADB repository
+
+    `git clone https://github.com/projectatomic/adb-atomic-developer-bundle.git`
+
+ 1. Run OpenShift instance in ADB
+
+    - `cd adb-atomic-developer-bundle/components/centos/centos-openshift-setup`
+    - `vagrant up`
+    - `vagrant ssh`
+
+ 1. Add Docker Registry on your local machine as insecure
+     
+    - `sudo vi /etc/sysconfig/docker`
+    - Uncommend INSECURE_REGISTRY line and add your local Docker Registry (`'--insecure-registry=10.1.2.1:5000'`)
+    - `sudo systemctl restart docker`
+    - `export MY_REGISTRY=10.1.2.1:5000`
+    
+ 1. Deploy MLB Parks sample application on OpenShift 
+
+    - `oc login` (username: openshift-dev password: devel)
+    - `oc new-project mlbparks`
+    - `oc create -f https://raw.githubusercontent.com/gshipley/openshift3mlbparks/master/mlbparks-template.json`
+    - `oc new-app mlbparks`
+    - `oc get builds -w`
+    - Wait for mlbarks build to complete.
+
+        ```
+        NAME         TYPE      FROM          STATUS     STARTED         DURATION
+        mlbparks-1   Source    Git@d63267b   Complete   4 minutes ago   4m6s
+        ```
+
+
+## Use Openshift2Nulecule to export sample application
+ 1. Install Openshift2Nulecule
+    - Add ADB testing repo (required only for ADB 1.7.0)
+        ```
+        sudo sh -c "cat > /etc/yum.repos.d/centos7-adb-cbs-test.repo <<EOF
+        [centos7-adb-cbs-test]
+        name=CentOS-$releasever - ADB CBS Testing
+        baseurl=https://cbs.centos.org/repos/atomic7-adb-common-testing/x86_64/os/
+        gpgcheck=0
+        enabled=1
+        EOF
+        "
+        ```
+
+    - install openshift2nulecule
+
+       `sudo yum install openshift2nulecule`
+
+ 1. Get address of OpenShift internal Docker registry 
+     
+    ```
+    export OC_REGISTRY=$(sudo oc --config /var/lib/origin/openshift.local.config/master/admin.kubeconfig get svc docker-registry -o template --template="{{ .spec.clusterIP }}"):5000
+    ```
```

Comment:
Typo in path to `kubeconfig`, correct path is `/var/lib/openshift/openshift.local.config/master/admin.kubeconfig`